mindsdb/interfaces/knowledge_base/controller.py

```diff
@@ -124,10 +124,11 @@ def insert(self, df: pd.DataFrame):
 
         # add embeddings
         df_emb = self._df_to_embeddings(df)
+        df = pd.concat([df, df_emb], axis=1)
```

Comment:
@tmichaeldb perhaps we should change `cohere_handler` to be in line with other embedding handlers rather than kb?

As currently kb works with langchain_embeddings, open_ai and also hf embeddings